Makefile

```diff
@@ -0,0 +1,94 @@
+# Package configuration
+PROJECT = sandbox-ce
+
+# superset upstream configuration
+SUPERSET_REPO = https://github.com/apache/incubator-superset.git
+SUPERSET_VERSION = release--0.32
```

Comment:
`0.32.0rc2`

Reply:
this was changed during the project rename? Because I copied the values from `superset-compose` master.

Reply:
yep: https://gist.github.com/smacker/f0a317d5e618eedbc5b77ba36cad1f0c#file-move_superset-txt-L13
They actually point on the same commit most probably but still